src/drm.cpp

```diff
@@ -1258,9 +1279,26 @@ drm_prepare_liftoff( struct drm_t *drm, const struct FrameInfo_t *frameInfo )
 				crtcW = crtcH;
 				crtcH = w;
 			}
-
-			liftoff_layer_set_property( drm->lo_layers[ i ], "rotation", g_bRotated ? DRM_MODE_ROTATE_270 : DRM_MODE_ROTATE_0);
-
+      switch ( g_drmModeOrientation )
```

Comment:
Style: we use tabs for indentation.

Reply:
You monster! I'll fix it up.

Reply:
Replaced spaces for tabs and re-indented